Agents/ResultedConsumptionCalculationAgent/README.md

```diff
@@ -0,0 +1,216 @@
+## Description
```

Comment:
I think something that is missing at the start of each of the readme files is a statement to give some broader context. For example, that the agent was created alongside agents X, Y and Z to support the analysis of the impact of heat pumps in the UK (you could potentially cite your and Tom's preprints).